src/Table/Cell.jsx

```diff
@@ -0,0 +1,36 @@
+import React, {PropTypes} from "react";
+
+import {joinClasses as cls} from "../utils";
+
+require("./Cell.less");
+
+
+export default function Cell({children, className, noWrap, width, ...additionalProps}) {
+  const {cssClass} = Cell;
+
+  return (
+    <td
+      className={cls(
+        cssClass.CELL,
+        noWrap && cssClass.NO_WRAP,
+        className
+      )}
+      style={{width}}
+      {...additionalProps}
+    >
+      {children}
+    </td>
+  );
+}
+
+Cell.propTypes = {
+  className: PropTypes.string,
+  children: PropTypes.node,
+  noWrap: PropTypes.bool,
+  width: PropTypes.string,
```

Comment:
If we want to be precise about the propTypes we can actually custom validate ones like this to be valid CSS dimensions.